test/cli/daemon.js

```diff
@@ -91,6 +91,33 @@ describe('daemon', () => {
     }).catch(err => done(err))
   })
 
+  skipOnWindows('should handle API Array and Gateway Array', function (done) {
```

Comment:
hey @alanshaw I'm not sure if you wanted this to be tested in another location as well.